src/tests/models/manager.js

```diff
@@ -16,9 +16,9 @@ module.exports = {
 async function upsertTest(testRawData, existingTestId) {
     const testArtilleryJson = await testGenerator.createTest(testRawData);
     let id = existingTestId || uuid();
-    testRawData.fileId = await fileManager.createFileFromUrl(testRawData);
+    const fileId = await fileManager.createFileFromUrl(testRawData);
```

Comment:
```suggestion
    let fileId;
    if (testRawData['processor_file_url']){
         fileId = await fileManager.createFileFromUrl(testRawData['processor_file_url']);
    }
```

Reply:
+ refactor `createFileFromUrl` to accept only the `processor_file_url` because that is all it needs